docs/config_file.md

```diff
@@ -0,0 +1,14 @@
+# Kops config file
+
+Kops will look in the following locations for a config file, in order of priority:
```

Comment:
Questions

1. What can go into the file?  What can we override with this config file?
2. Can we override environment variables with this file?